cmd/clairctl/clair/analyze.go

```diff
@@ -0,0 +1,32 @@
+package clair
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+
+	"github.com/coreos/clair/api/v1"
+)
+
+//Analyze get Analysis os specified layer
```

Comment:
```go
// Comments in Go should look like this.
```

Full sentences with period, a space after the double slashes, and wrapped to 80 chars.